frontend/admin/src/models.ts

```diff
@@ -22,9 +22,9 @@ export type User = {
 
 export type Room = {
   id: string
-  roomNumber: string
-  city: string
-  building: string
+  name: string
+  cityId: string
+  buildingId: string
```

Comment:
Depending on how the BE will return this data it can have fields for `building` and `city` as well.